tools/logos_check.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
+
+import sys
+import argparse
+from pathlib import Path
+
+from PIL import Image
+
+
+class ImageCheck():
+    def __init__(self, imgpath: Path) -> None:
+        self.imgpath = imgpath
+        self.image = Image.open(imgpath)
+
+        self.fails: list[str] = []
+
+    def check(self) -> bool:
+        self.fails.clear()
+        for fn_name in dir(self):
+            if fn_name.startswith("check_"):
+                getattr(self, fn_name)()
+
+        if not self.fails:
+            return True
+
+        print(f"Image '{self.imgpath}' failed tests:")
+        for fail in self.fails:
+            print(f"  - {fail}")
+        print()
+        return False
+
+    def check_type(self) -> None:
+        format = self.image.format.lower()
+        accepted_formats = ["png", "jpeg", "webp"]
+        if format not in accepted_formats:
+            self.fails.append(f"Image should be one of {', '.join(accepted_formats)} but is {format}")
+
+    def check_dimensions(self) -> None:
+        dim_min, dim_max = 96, 300
```

Comment:
Could we maybe hardcode those constants at the top of the file for easier changes?